src/main/java/com/example/backendspringboottechiteasycontroller/exceptions/TVNameTooLongException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.backendspringboottechiteasycontroller.exceptions;
```

Comment:
You haven't yet used this exception or the other exception in you code. You can use it like `throw new TvNameTooLongException("This name is too long")`